src/windows/models/properties_model.py

```diff
@@ -568,6 +571,7 @@ def value_updated(self, item, interpolation=-1, value=None, interpolation_detail
             # Save changes
             if clip_updated:
                 # Save
+                c.data = clip_data
```

Comment:
`clip_data` is updated heavily before this moment - but never added back to the Clip object before saving